apps/docker/Dockerfile

```diff
@@ -40,27 +40,38 @@ RUN addgroup --gid $GROUP_ID acore && \
     passwd -d acore && \
     echo 'acore ALL=(ALL:ALL) NOPASSWD: ALL' >> /etc/sudoers
 
+# must be created to set the correct permissions on them
+RUN mkdir -p /azerothcore-internal/env/dist/data
+RUN mkdir -p /azerothcore-internal/env/dist/logs
+RUN mkdir -p /azerothcore-internal/env/dist/etc
+RUN mkdir -p /azerothcore-internal/var/build
```

Comment:
```suggestion
RUN mkdir -p /azerothcore-internal/var/build/obj
```

Reply:
got this error while testing:
![image](https://user-images.githubusercontent.com/4015312/116006201-f81dbd80-a5be-11eb-89b6-270508bf7cc3.png)